polkadot/service/src/lib.rs

```diff
@@ -433,15 +484,7 @@ impl<B, E> Service<B, E>
 		let barrier = ::std::sync::Arc::new(Barrier::new(2));
 		on_demand.map(|on_demand| on_demand.set_service_link(Arc::downgrade(&network)));
 
-		let mut maybe_stats = ws::ClientBuilder::new("ws://127.0.0.1:1024").ok()
-			.and_then(|mut x| x.connect_insecure().ok());
-		if let Some(ref mut stats) = maybe_stats {
-			let msg = format!("{{\"_type\": \"init\", \"version\": \"{}\", \"chain\": \"{:?}\", \"best\": {}}}",
-				crate_version!(),
-				config.chain_spec,
-				best_header.number);
-			let _ = stats.send_message(&ws::Message::text(msg));
-		}
+		slog_info!(SLOG_ROOT, "Starting version {version} (height is {best})", version = crate_version!(), best = best_header.number; "chain" => format!("{:?}", config.chain_spec));
```

Comment:
It would probably make sense to get the binary name in addition to the actual version in here as well, so we allow for multiple implementations to report to the same endpoint.